app/controllers/feedback_forms_controller.rb

```diff
@@ -27,6 +29,9 @@ def create
         format.html do
           redirect_to params[:url]
         end
+        format.turbo_stream do
+          render json: flash
```

Comment:
This is odd. They are requesting a turbo-stream response, but you are returning json instead.